### PR TITLE
Avatar is saving to DB and retrieving on subsequent logins

### PR DESCRIPTION
**Pull request by sbarkerdudley**

### CODE REVIEW

File: server/controllers/jobSearchController.js [hunk 1]
```diff
@@ -59,9 +59,9 @@ exports.jobSearch = (req, res) => {
 
   } = req.query;
 
-  //Add the input experience level to keywords
-  //keywords = experienceLevel && `${experienceLevel} ${keywords}`;
-  keywords = `${experienceLevel} ${keywords}`;
+  if (keywords && experienceLevel) {
+    keywords = `${experienceLevel} ${keywords}`;
```
Comment (Gobleizer) [inline server/controllers/jobSearchController.js:63]: Good catch here

File: server/models/userModel.js [hunk 2]
```diff
@@ -16,12 +16,12 @@ const user = {
...
   },
   uploadPDF: async (url, fileUse, id) => {
-    return pool.query(`UPDATE Users SET ${fileUse}_pdf_url = $1
+    return pool.query(`UPDATE Users SET ${fileUse}_url = $1
       WHERE id=$2;`,
     [url, id])
   },
   getPDF: (fileUse, id) => {
-    return pool.query(`SELECT ${fileUse}_pdf_url FROM Users WHERE id=$1`, [id]);
+    return pool.query(`SELECT ${fileUse}_url FROM Users WHERE id=$1`, [id]);
```
Comment (Gobleizer) [inline server/models/userModel.js:24]: This change will require any other controller using getPDF to be refactored to match. I can take care of this, but it is an important consideration.

File: server/controllers/usersController.js [hunk 3]
```diff
@@ -12,8 +11,8 @@ module.exports.insertPDF = (req, res) => {
 
 module.exports.getPDF = (req, res) => {
   const { fileUse } = req.params;
-  const userId = req.user.id;
+  const userId = req.user.id || 1;
```
Comment (Gobleizer) [inline server/controllers/usersController.js]: Defaulting to 1 here is not preferable. Instead we should kick back an unauthorized error. See the save jobs or applied jobs controllers.

**Review by Gobleizer — CHANGES_REQUESTED** [review]
Please fix the getPDF Controller. I will fix the one click application after this is merged to match the changes to "fileUse" parameter.

File: server/controllers/usersController.js (hunk 3 above)
Comment (Gobleizer) [reply server/controllers/usersController.js]: Example snippet in Slack now

File: server/models/userModel.js (hunk 2 above)
Comment (sbarkerdudley) [reply server/models/userModel.js:24]: I changed everything as required and it's working for both PDF and avatar image uploads now

File: server/controllers/usersController.js (hunk 3 above)
Comment (sbarkerdudley) [reply server/controllers/usersController.js]: Just error fixing on my end to proceed with other changes. Feel free to handle however

**Pull request closed without merging**